doc/performance-investigations/0021-upgrade-python-to-3.10.8/SUMMARY.md

```diff
@@ -0,0 +1,33 @@
+# Python 3.10.8 Upgrade investigation
+
+EQ Questionaire Runner is currently on `3.9` .
+This investigation is to see the performance of Runner when upgraded to `3.10.8` compared to `3.9` .
+
+https://github.com/ONSdigital/eq-questionnaire-runner/tree/upgrade-python-3.10.8
+
+## Benchmark profile
+
+| Option | Value |
+|--------|-------|
+| Requests file | requests/test_benchmark_business.json|
+| Run time | 20m |
+| User wait time minimum | 1s |
+| User wait time maximum | 2s |
+| Clients | 50 |
+| Hatch rate | 50 |
+
+## Results
+
+Results based on 99th percentile timings.
+| Metric              | Baseline | Investigation |
+| ------------------- | -------- | ------------- |
+| Questionnaire GETs  | 469ms    | 433ms         |
+| Questionnaire POSTs | 306ms    | 186ms         |
+| All requests        | 409ms    | 341ms         |
+| Total Requests      | 65,857   | 65,678        |
```

Comment:
The numbers don't really make sense. How can Investigation be faster by serve less requests?.

Reply:
I am not sure what you mean , could you explain it to me. I can re run the benchmarks again for both investigation and baseline to see if the results are different 👍

Reply:
As in, the numbers suggest that the investigation (3.10) requests are faster, but it ended up serving less total requests which doesn't make sense?. If the requests were faster then, the Total Requests should have been higher right?.

Reply:
Oh I see, that is strange. 
I shall re run the benchmark for a longer duration rather then the default 20 minutes and get more accurate averages.